templates/download.html

```diff
@@ -0,0 +1,194 @@
+{% extends "_base/base.html" %}
+{% block title %}Ubuntuを入手する | Ubuntu{% endblock %}
+{% block meta_description %}Ubuntuは、スマートフォン、タブレット端末、PCからサーバー、クラウドまであらゆる環境で動作するオープンソースのソフトウェアプラットフォームです。{% endblock %}
+
+{% block outer_content %}
+<div id="main-content" class="u-no-margin--top">
+  <section class="p-strip--light is-deep">
```

Comment:
I think this looks better:
```suggestion
  <section class="p-strip--light is-deep is-bordered">
```